cluster/images/etcd/migrate/versions.go

```diff
@@ -187,11 +179,3 @@ func ParseSupportedVersions(list []string) (SupportedVersions, error) {
 	return versions, nil
 }
 
-// MustParseSupportedVersions parses a comma separated list of etcd versions or panics if the parse fails.
-func MustParseSupportedVersions(list []string) SupportedVersions {
```

Comment:
weird, this is being used in a test case, so please don't delete:
https://cs.k8s.io/?q=MustParseSupportedVersions&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes